src/components/caption-list/captionList.tsx

```diff
@@ -104,10 +104,12 @@ export class CaptionList extends Component<Props> {
 
   render() {
     const {data} = this.props;
+    let isSearchCaption = false;
```

Comment:
define if ref should be set inside `_getCaptionProps` method (for searchCaption or highlightedCapiton) and set ref if the property is true.